templates/taxi/car_list.html

```diff
@@ -1,13 +1,24 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>Car list <a href="{% url "taxi:car-create" %}">+</a></h1>
   {% if car_list %}
     <ul>
       {% for car in car_list %}
         <li>
-          <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
-          {{ car.model }} ({{ car.manufacturer.name }})
+          <tr>
+            <td>
+              <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
+          {{ car.model }} ({{ car.manufacturer.name }})    
+            </td>
+            <td>
+              <a href="{% url "taxi:car-update" pk=car.id %}">Update</a>
+            </td>
+            <td>
+              <a href="{% url "taxi:car-delete" pk=car.id %}">DELETE</a>
+            </td>
+          </tr>
```

Comment:
The use of `<li>` and `<tr>` tags together is incorrect as they belong to different types of HTML structures. Consider using either a `<ul>` or `<table>` structure consistently. If you choose a table, replace `<ul>` with `<table>` and `<li>` with `<tr>`.